WIP: Missions to unlock the Bactrian

tehhowch opened the pull request:
@Amazinite

This will let us keep track of commentary on the current state of the content, rather than comment on incremental diffs from branch commits. Your master was out-of-date so... 🗡 💥

Amazinite committed:
Vomitting my notes out onto the world

deep missions.txt contains a bunch of notes and skeleton missions for
how the Deep missions will be written and play out for the player to be
able to unlock the Bactrian.

Right now this is just a load of vomit that might be hard to follow.
I'll start filling in the holes tomorrow, starting (although out of
order) with the Deep: Remnant string of missions (since I already have
in mind how the entire string will go), then working my way back to
Deep: TMBR and Deep: Convoy.

A rewrite of Deep: Star Queen Rescue will be the final thing.

Amazinite committed:
Adding the license requirement to the Bactrian

Amazinite committed:
Adding the first half of Deep: Remnant

Amazinite committed:
Typo fix

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1121 @@
+	name `Fetching Keystones`
+	description `Buy 10 Quantum Keystones from Hai space and return them to <destination> so that Ivan may research them and their link to the spacial anomaly in Terminus.`
+	source
+		attributes deep
```
Can offer on Valhalla, which makes the bit where Ivan leaves for his lab on Valhalla odd (since you'd logically go there otherwise). Probably a `distance / near` filter with min 1 max 100 as well as the `attributes deep`, so that it can't offer on Valhalla.

Amazinite replied:
Fixed in 103d80a

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1121 @@
+	npc
+		personality nemesis waiting
+		system
+			distance 3
```
I get what you're going for here, but I wonder if there is a better way to set it up. There's a pretty good chance these NPC fleets don't spawn in the desired systems. (e.g. pick chokepoints).

Of course, if tehhowch/endless-sky#3 merges, then these could be patrolling and the problem is solved

Amazinite replied:
That isn't actually what I was going for, but it may be good to do that anyways. Only question is should they be staying in those chokepoints or should they just spawn in chokepoints and then travel toward the player?

Edit: Going to work on Deep: Convoy and Deep: TMBR tomorrow (combined they both amount to what Deep: Remnant is, so I should be able to get them both done), so I'll tackle the situation then.

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1121 @@
+		
+event "reset arculus"
+	system Arculus
+		fleet "Large Remnant" 1500
```
`add fleet`

tehhowch reviewed (commented):
Not gonna suggest changes or tweaks to dialog until much closer to the end 👍

Amazinite committed:
Finishing Deep: Remnant

Amazinite committed:
Making sure Ivan doesn't leave Valhalla for Valhalla

Amazinite commented:
>Not gonna suggest changes or tweaks to dialog until much closer to the end

How about suggesting dialog changes once I finish a string (e.g. like how I just finished Deep; Remnant)? That way I can carry the feedback into the other missions instead of possibly making the same mistakes repeatedly. :)

Amazinite committed:
Changing the to offer conditions on some of the "start" Deep missions

The same number of jobs need to be done to advance to the "middle" Deep
mission, but now you can get some of the "start" Deep missions sooner.

The jobs you need to do relate to the missions. e.g. Deep: TMBR requires
the scientist related jobs because you're dealing with scientists. The
Deep: Convoy missions require mystery cube jobs because those cubes will
relate to the missions in some way. (This is also why Deep: Mystery
Cubes requires one convoy mission to be done.)

Amazinite committed:
Some changes and fixes

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1176 @@
+	on enter Arculus
+		dialog `This region of space appears to be inhabited. You should attempt to find out more about the beings living here so that you may tell Ivan.`
+	
+	on enter Nenia # Forces the player to check the Ember Waste, but the player could have taken a keystone and seen for themself that they could enter the Ember Waste while the group was doing research, therefore making this mission extraneous. Any possible solutions? Create a C branch for if the player did not know of the Remnant at the time of speaking with Ivan but later learned about them when  ...
```
There is a bit of similarity with the existing Remnant mission to scan void sprites, so perhaps we condition this mission to not offer if that has been done. If the player goes to explore the Waste while the Deep is researching the stones, they may stay there for some time before returning to the rest of human space (probably will return when the Remnant asks them to fetch Hai stones).
If this mission can't offer, then a C mission would be needed to set the relevant conditions that 1A would otherwise set, and have the player just tell Ivan about the interactions with the Void Sprites

Amazinite committed:
Deep: Convoy missions

Amazinite committed:
Done the first two parts of Deep: TMBR

Also allowed for some different options as to how you can reach "Deep
middle;" The player must do at least one convoy mission to unlock Deep:
Remnant 0, which will allow them to do Deep: Singularity without doing
2+ convoy missions, but should the player decline Deep: TMBR, they must
do all three convoy missions if they are to continue.

The scientist characters mentioned in Deep: TMBR will be used in Deep:
Remnant and Deep: Singularity, so Deep: TMBR now does not offer if
either of the two other strings are started.

tehhowch committed:
Tehhowch Edits Round 1 (#1)

* Edits Rnd1

* Removing changes I disagree with

Amazinite committed:
Added a C branch for the Remnant string

Amazinite commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1354 @@
+			`	Ivan looks like he is about to fall over with joy after you mention having entered the wormhole. "Really? Please, tell me what you found!"`
+			branch friendly
+				has "event: ember waste label"
+			`	You tell Ivan all that you saw on the other side of the wormhole, and how you found strange-looking humans who communicate in song and sign language, seemed obsessed with blood, and attacked you after you wouldn't let them have some of yours.`
```
Whatever changes occur on that Remnant log PR should change here as well (and in the other two locations in this string mentioning blood).

tehhowch replied:
I phrased it to be a bit of a comical hot-take. This player literally had to run away from some possible aliens who wanted to prod him with a strange device that collects blood. That they were then willing to kill him because he decided to not let strange devices near his body *is* a little obsessive 😆

Amazinite replied:
Do you want to handle the logs for these missions or should I? 
After all the missions are written, I'd do people logs for Paris, Ivan, Maria, maybe Laura (or just mention her in Paris' log), a faction log or two for Deep Security, as well as some normal logs on the last three missions strings (`Remnant`, `Singularity`, and `Scientist Rescue`) as well as `Mystery Cubes`,

tehhowch replied:
Logs can be saved until the end i think, when we know what to remind the player of

Amazinite committed:
Finished the last part of Deep: TMBR

Deep: Remnant 0 also now requires "First Contact: Hai" to be offered for
the situation where the player might get this mission without having
done Deep: TMBR

Also changed some dialog in Deep: Remnant Technology to make it more
clear that the player does not need to keep helping the scientists if
they wish not to.

Amazinite commented on data/deep missions.txt:
```diff
@@ -353,33 +353,45 @@ mission "Deep: TMBR 1"
+			`	The crowd gets larger and largerover time, and judging by the increased number of ships landing, word must have gotten out quickly that "There Might Be Riots" are back.`
+			`	Just as the concert begins with the band playing as loud as ever, you spot the scientists all in the front row, smiling and cheering as the music roars. The band starts off with one of the songs you heard them play in Hai space, and since none of the audience members are singing along it's clear that it is new to them.`
+			`	During an intermission, you invite the scientists onto your ship to personally meet the band. The thrill in the eyes of each of the scientists is astounding as they meet the members of "There Might Be Riots."`
+			`	Before the intermission ends, each of the scientists thank you personally for bringing the band to Midgard, and Isaac even hands you <payment>. "It's the least we could give you," he adds. As the scientists start to depart your ship to listen to the rest of the concert, Hannah kisses you on the cheek without a word before running off with the crowd.`
```
I was feeling cheeky.

tehhowch replied:
Not sure romantic undertones are going to make it to upstream. I think this and the previous block can be combined somehow as well.

Amazinite replied:
Yeah, I'll change it. Wasn't really feeling it while writing this one. (Hence why I've also been stalling on doing the other two strings.)

Amazinite replied:
I'll change it to a bear hug.

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+		not "Deep: Singularity"
+	on offer
+		conversation
+			`When you enter the <planet> spaceport, you hear what is undoubtedly a "There Might Be Riots" song playing loud enough for anyone in the spaceport to hear. You look around and find that the source of the music is the external speakers of a Bounder that is surrounded by a few people dancing.`
```
\`When you enter the \<planet\> spaceport, you hear what is undoubtedly a "There Might Be Riots" song, playing loudly enough for everyone in the spaceport to hear. A quick glance around reveals the source of the music to be the external speakers of a nearby Bounder, which is surrounded by a few people dancing.\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+				`	(Ignore them.)`
+					defer
+					
+			`	You grow nearer to the Bounder and count five people dancing in total: three women and two men. One of the woman dancing notices you walking over. "It looks like someone else is about to join the party!" This is met by a collective cheer from the rest of the dancers.`
```
\`	You approach the Bounder and count five people dancing in total: three women and two men. One of the women dancing notices you walking over and exclaims, "Looks like someone else is about to join the party!" Her shout is met by a raucous cheer from the rest of the dancers, and they pull you into the fray.\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+					defer
+					
+			`	You grow nearer to the Bounder and count five people dancing in total: three women and two men. One of the woman dancing notices you walking over. "It looks like someone else is about to join the party!" This is met by a collective cheer from the rest of the dancers.`
+			`	A few minutes of dancing later, one of the people introduces himself to you as Garrison. He lists off the names of the rest of the people with him; Isaac, Maria, Hannah, and Larua, then explains that they are all Deep scientists. They are currently enjoying a short vacation listening to their favorite band before they must travel to Midgard to continue their work for the Deep.`
```
\`	After a few minutes of dancing, the music begins to subside. One of the dancers introduces himself to you as Garrison, and lists off the names of his fellow dancers; Isaac, Maria, Hannah, and Laura. As you make your introductions, you learn that everyone here is part of the same research team, working for one of the Deep's many science bureaus, and they are currently wrapping up a short vacation by listening to their favorite band before they travel to Midgard on assignment.\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+					
+			`	You grow nearer to the Bounder and count five people dancing in total: three women and two men. One of the woman dancing notices you walking over. "It looks like someone else is about to join the party!" This is met by a collective cheer from the rest of the dancers.`
+			`	A few minutes of dancing later, one of the people introduces himself to you as Garrison. He lists off the names of the rest of the people with him; Isaac, Maria, Hannah, and Larua, then explains that they are all Deep scientists. They are currently enjoying a short vacation listening to their favorite band before they must travel to Midgard to continue their work for the Deep.`
+			`	"How do you know about 'There Might Be Riots?'" Hannah asks you.`
```
TMBR may be popular enough that this is like asking how someone knows about Elton John or Elvis

Amazinite replied:
Lol, who's Elton John?

Jokes aside, whether TMBR are popular enough or not, Hannah assumes you know the artist of the songs that are playing in the first place. The player could have just been some rando who wanted to join in on a party, so how the scientists are lead to knowing that the player knows TMBR should be changed in general.

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+					goto met
+				`	"I just like to listen to their music."`
+			
+			`	One of the other scientists, Issac, looks right at you and stares. After a few moments he says "I know you. I'm fairly sure I saw you on the 'Songs for the End of Civilization' broadcast from Pilot." The rest of the scientists start agreeing with him as they look at you closer.`
```
\`	One of the other scientists, Issac, looks right at you and stares. After a few moments he says, "I know you! You were in the 'Songs for the End of Civilization' broadcast from Pilot!" The rest of the scientists start agreeing with him as they look at you closer.\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+			`	One of the other scientists, Issac, looks right at you and stares. After a few moments he says "I know you. I'm fairly sure I saw you on the 'Songs for the End of Civilization' broadcast from Pilot." The rest of the scientists start agreeing with him as they look at you closer.`
+			
+			label met
+			`	"You know 'There Might Be Riots?' You are so lucky!" say Maria.`
```
say -> says

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+			
+			label met
+			`	"You know 'There Might Be Riots?' You are so lucky!" say Maria.`
+			`	"I couldn't even see them during their 'Trouble on the Tarmac Tour,' but you've met them! That's insane." Laura cries out.`
```
\`	"I couldn't even see them during the 'Trouble on the Tarmac' tour, but you've met them?! That's insane!" cried Laura. "That album had some of my favorite songs."\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+			label met
+			`	"You know 'There Might Be Riots?' You are so lucky!" say Maria.`
+			`	"I couldn't even see them during their 'Trouble on the Tarmac Tour,' but you've met them! That's insane." Laura cries out.`
+			`	"Which makes me wonder," Hannah chimes in. "Do you know where the band is? They've been missing for a while now, ever since that broadcast on Pilot. I've heard rumors that the Republic detained them, but since you're here then you must know where they are."`
```
\`	Hannah chimes in, "I've been wondering: Do you know where the band is? They've been quiet for a while now, ever since that broadcast. I heard rumors that the Republic detained them for trespassing, but since you're here..."\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+			`	"Which makes me wonder," Hannah chimes in. "Do you know where the band is? They've been missing for a while now, ever since that broadcast on Pilot. I've heard rumors that the Republic detained them, but since you're here then you must know where they are."`
+			choice
+				`	"I know where they are, and rest assured that they're safe."`
+				`	"Sorry, I don't know where they are. I need to leave now."`
```
Seems a little terse, and almost like it should prompt other dialog rather than a `decline`

Amazinite replied:
Agreed; how about the scientists saying goodbye since you just danced with them and all?

Amazinite replied:
Added a `goto decline` after this choice where the scientists are disappointed and say goodbye.

tehhowch replied:
Stripping the `I need to leave now.` makes sense, given the decline branch text.

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+				`	"I know where they are, and rest assured that they're safe."`
+				`	"Sorry, I don't know where they are. I need to leave now."`
+					decline
+			`	The group falls silent until Maria asks "Do you think you could bring them to us for a live concert?"`
```
\`	The group falls silent until Maria asks, "Could you bring them to us for a live concert?"\`

Maybe mention some sort of benefit that the Deep can provide in exchange for the concert - I imagine plenty of folks ask famous bands for concerts, to the point that the requests would be basically denied no matter the relationship between the band and the asking party. Perhaps there is some recent development in the Deep that builds on the TMBR message from the Pilot broadcast?

Amazinite replied:
I'll have to think of the reasoning.

Amazinite replied:
Any ideas other than `we can pay`?

Perhaps the research these scientists are working on involves trying to lower the amount of weapons of war. (How would the singularity do that, though, since this is what I'm thinking they're working on.)

tehhowch replied:
Could be a MAD style logic... make war unlikely because of the insane destruction it would cause to all involved.

That's likely too 20th century to make sense though.
Thought: locals of the Deep/Republic in general are upset about the role the Deep has had in furthering war efforts (requires electron laser developments to make sense), and thus the Deep wants to show some collaboration with known anti-war groups like TMBR to help quell this science-for-war perception.

The Deep would be able to offer a more guaranteed communications channel than what is available after Pilot, so the TMBR message would be easier to send to all the reaches of the Republic(/FW). If we condition for some level of FW/Syndicate/Republic story, the origin of the broadcast being the Deep (rather than a core Republic/Syndicate/FW world) may help showcase the message as organic and not forced propaganda from a 'captured' TMBR. Though the Deep is already 'known' as highly associated with the Republic, so the perception could be the opposite.

We can also use the TMBR stage manager Becca - perhaps she has been wondering where her best clients have gone, and she happens to be in the Deep.

Amazinite replied:
Becca actually seems to always be with the band; in all TMBR missions you transport 8 people.

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+			`	The group falls silent until Maria asks "Do you think you could bring them to us for a live concert?"`
+			choice
+				`	"I'd love to."`
+				`	"No thanks. I have better things to do."`
```
Could have a third option about the band probably not wanting to do that / maybe being too busy or not ready, and set a condition in that option's decline that opens up a new mission branch (in case there isn't enough content to add yet 😆 ).

Amazinite replied:
Lol, maybe.

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+				`	"No thanks. I have better things to do."`
+					decline
+			
+			`	All five scientists burst out in a cheer, clearly excited that they will be able to see their favorite band live. After the cheering dies down, Laura says "We must get going now, Captain. We'll be on Midgard for the next few months, so if you find 'There Might Be Riots,' please bring them there." Each member of the group says goodbye to you as they board their Bounder and leave for Midgard.`
```
\`	All five scientists burst out in a cheer, clearly excited for the chance to see their favorite band live. As the cheering dies down Laura says, "We must get going now, \<first\>. We're being stationed on Midgard for the next few months, so if you do find 'There Might Be Riots,' please bring them there." Each member of the group says goodbye to you as they depart.\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+					decline
+			
+			`	All five scientists burst out in a cheer, clearly excited that they will be able to see their favorite band live. After the cheering dies down, Laura says "We must get going now, Captain. We'll be on Midgard for the next few months, so if you find 'There Might Be Riots,' please bring them there." Each member of the group says goodbye to you as they board their Bounder and leave for Midgard.`
+			`	If you are going to look for the band "There Might Be Riots," you should go to <destination> where you last saw them.`
```
\`\`
\`	(If you are going to look for the band "There Might Be Riots," you should go to \<destination\>, where you last saw them.)\`

Amazinite replied:
Ehh, not sure about the 4th wall breaking.

tehhowch replied:
I mean, i just parenthesized the text you added first :)

We can totally drop the hint and add a `to complete` block that has one of the later missions done as a condition, and then use a low random chance mission offered only in Hai space that can trigger the conversation and mission-with-a-destination once this mission is active.
So, there'd be two concurrent missions, once that says "Bring TMBR back to the Deep", and the Hai-space mission would trigger and say "Find TMBR on \<planet\>".

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+		has "Deep: TMBR 0: done"
+	on offer
+		conversation
+			`It takes you a while, but after a few hours of asking around you finally find someone who knows where the band are. "I saw them on <destination>," the human says. "They were throwing a pretty narly concert just last week. I haven't heard of them playing anywhere else, so they might still be there."`
```
\`It takes you a while, but after a few hours of asking around you finally find someone who knows where the band is. "I saw them on \<destination\>," she says. "They were throwing a pretty gnarly concert just last week. I haven't heard of them playing anywhere else, so they might still be there."\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+		has "Deep: TMBR 1: done"
+	on offer
+		conversation
+			`You find "There Might Be Riots" playing a gig right in the spaceport of <planet>. Judging my the large amount of Hai in the audience, they seem to have amassed quite the following in Hai space.`
```
\`You find "There Might Be Riots" playing a gig right in the spaceport of <planet>. Judging by the large number of Hai in the audience, they seem to have amassed quite the following, even in Hai space.\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+	on offer
+		conversation
+			`You find "There Might Be Riots" playing a gig right in the spaceport of <planet>. Judging my the large amount of Hai in the audience, they seem to have amassed quite the following in Hai space.`
+			`	You wait for the band to finish their concert before speaking with Ulrich. They play a few songs you haven't heard before; the band must have written a few while staying in Hai space. Once the concert ends and Ulrich walks off the makeshift stage that has been set up on a landing pad, you walk over to him.`
```
\`	While the band intermixes crowd favorites and some new songs clearly inspired by their time in Hai space, you make your way to the side of the makeshift stage. Once the concert ends, Ulrich notices you and approaches.\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+		conversation
+			`You find "There Might Be Riots" playing a gig right in the spaceport of <planet>. Judging my the large amount of Hai in the audience, they seem to have amassed quite the following in Hai space.`
+			`	You wait for the band to finish their concert before speaking with Ulrich. They play a few songs you haven't heard before; the band must have written a few while staying in Hai space. Once the concert ends and Ulrich walks off the makeshift stage that has been set up on a landing pad, you walk over to him.`
+			`	"Captain <last>! It's been a while. We've been having a great time with the peaceful squirrels ever since you dropped us off here. It's the craziest experience we've ever had." Ulrich goes on to explain a number of adventures that the band has been on since they came to Hai space, and how they were all surprised that even in such a peaceful place like this there are still wars. He then ends  ...
```
\`	"Captain <last>, it's been a while! We've been having a great time with the peaceful squirrels ever since you dropped us off. It's the craziest experience we've ever had." He explains a number of adventures that the band has been on since they came to Hai space, and how they were all surprised that even in such an idyllic place war was still a reality. "Say, what brings you here?"\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+			`You find "There Might Be Riots" playing a gig right in the spaceport of <planet>. Judging my the large amount of Hai in the audience, they seem to have amassed quite the following in Hai space.`
+			`	You wait for the band to finish their concert before speaking with Ulrich. They play a few songs you haven't heard before; the band must have written a few while staying in Hai space. Once the concert ends and Ulrich walks off the makeshift stage that has been set up on a landing pad, you walk over to him.`
+			`	"Captain <last>! It's been a while. We've been having a great time with the peaceful squirrels ever since you dropped us off here. It's the craziest experience we've ever had." Ulrich goes on to explain a number of adventures that the band has been on since they came to Hai space, and how they were all surprised that even in such a peaceful place like this there are still wars. He then ends  ...
+			`	You explain to Ulrich the situation of the Deep scientists who are huge fans of "There Might Be Riots" and wish to see the band live, and how there are rumors floating around in human space as to the fate of the band after the broadcast on Pilot. "Oh yeah, I hadn't thought about what our fans would think after just disappearing like that. Maybe it's time to go back to the land of the humans. ...
```
What's their situation? A couple of adults who have a favorite band and like concerts? 😆 

\`	He listens intently to your story of the scientists on Midgard, but his brow furrows when you mention the rumor of imprisionment by the Republic.\`
\`	"Perhaps we've been away long enough. I'd have thought there would be some rumors or recordings that made it back to the Republic from here, but this place is a fairly well-kept secret so perhaps not. Maybe it's time to go back to the land of the humans."\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+	on complete
+		payment 400000
+		conversation
+			`You quickly draw a crowd around your ship after "There Might Be Riots" step off and begin setting up their equipment. While the band gets ready, you contact the spaceport authorities and have them notify Garisson, Isaac, Hannah, Maria, and Laura that the band is currently in the spaceport.`
```
\`Almost as soon as "There Might Be Riots" steps off your ship, a crowd gathers. When it is clear that the band is not simply disembarking, but preparing to perform, the uproar makes you glad that you had not given any advance notification to the locals. As the band gets ready, you contact the spaceport authorities and have them notify Garrison and the other scientists that you have returned. The authorities don't seem too enthused about hosting a concert with no advance warning, but they agree to send your message.\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+		payment 400000
+		conversation
+			`You quickly draw a crowd around your ship after "There Might Be Riots" step off and begin setting up their equipment. While the band gets ready, you contact the spaceport authorities and have them notify Garisson, Isaac, Hannah, Maria, and Laura that the band is currently in the spaceport.`
+			`	The crowd gets larger and largerover time, and judging by the increased number of ships landing, word must have gotten out quickly that "There Might Be Riots" are back.`
```
larger over

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+		conversation
+			`You quickly draw a crowd around your ship after "There Might Be Riots" step off and begin setting up their equipment. While the band gets ready, you contact the spaceport authorities and have them notify Garisson, Isaac, Hannah, Maria, and Laura that the band is currently in the spaceport.`
+			`	The crowd gets larger and largerover time, and judging by the increased number of ships landing, word must have gotten out quickly that "There Might Be Riots" are back.`
+			`	Just as the concert begins with the band playing as loud as ever, you spot the scientists all in the front row, smiling and cheering as the music roars. The band starts off with one of the songs you heard them play in Hai space, and since none of the audience members are singing along it's clear that it is new to them.`
```
\`	As Ulrich and the band take the stage, you spot the scientists in the front row, grinning and cheering wildly. The first song is one you heard them play in Hai space, and even though none in the audience know the words to sing along, they appear to love the message, and the cheers and screams only grow more frenetic.\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+
+mission "Deep: Remnant 2A"
+	landing
+	name `Scanning Remnant`
```
`Analyze Remnant Ships`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+
+mission "Deep: Remnant 1B"
+	landing
+	name `Scanning Remnant`
```
Analyze Remnant Ships

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+		has "knew of stones before scientists"
+	on offer
+		conversation
+			`You contact Ivan after you land and he sends you directions to his research lab, which is located on the outskirts of the spaceport city. When you arrive you are greeted by Ivan, Maria, and a number of other scientists on the team. Ivan begins to inform you on what their research has revealed.`
```
\`You contact Ivan after you land and he sends you directions to his research lab, located on the outskirts of the city. When you arrive you hardly have time to greet Ivan, Maria, and the others before Ivan begins discussing the results of their research.\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+			`	You tell Ivan about how the region of space beyond the wormhole is known as the "Ember Waste" and that it is inhabited by a group of friendly humans who fled there during the Alpha Wars. These humans have advanced independently from the rest of humanity for centuries. You mention their strange culture, sharing what you can remember about the songs the Remnant people sang.`
+			
+			label end
+			`	"This is very interesting, and would turn out to be an amazing discovery if true. Please, bring us the Quantum Keystones so that we might try to understand the properties which allow entry into the wormhole's threshold. Later we will learn more about these inhabitants you describe."`
```
add a bit about paying for some of the cost

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+		conversation
+			`You contact Ivan after you land and he sends you directions to his research lab, which is located on the outskirts of the spaceport city. When you arrive you are greeted by Ivan, Maria, and a number of other scientists on the team. Ivan begins to inform you on what their research has revealed.`
+			`	"We've analyzed the Quantum Keystones, and they do allow entry into the wormhole in Terminus just as you said. It would appear that the stones stabilize the wormhole as the ship passes through it, or maybe it is stabilizing the ship in some way and not the wormhole. We'll need much more time to fully understand the true mechanism that is at play here. Whatever the case is, one stone is requi ...
+			`	Ivan then asks you to fill the rest of the scientists in on what you have found on the other side of the wormhole. After you finish talking, they begin to chatter among themselves, seeming to be most interested in the human beings which inhabit the region.`
```
The B branch is for a player that has already met the Remnant, and has already told Ivan this info.

\`	"As you mentioned before, a single Quantum Keystone is sufficient to enter the wormhole. We've determined that there is a stabilization effect, either on the ship or the wormhole, that allows matter to cross the threshold. However, we'll need much more time to fully understand the true mechanism." Ivan asks, "Last time, you mentioned that there were humans living in this space, the... the 'Remainder'?"\`
\`	"Remnant."\`
\`	"Right, that was it. Well, we've decided that we need to know more about their ships and technology - perhaps they have some alternative method of interacting with these unstable wormholes that can accelerate our research."\`

Amazinite replied:
Love it.

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+		event "reset arculus"
+		event "deep: scan log research" 21
+		dialog
+			`The team is visibly excited when you return with the scanner logs. "Give us a few weeks to look these over," Ivan says, "then we will contact you about what we want you to do next."`
```
\`The team is visibly excited when you return with the scanner logs. "Give us a few weeks to look these over," Ivan says. "Then we will contact you about what we want you to do next."\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+
+mission "Deep: Remnant 1C"
+	landing
+	name `Scanning Remnant`
```
Analyze Remnant Ships

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+		has "event: deep: scan log research"
+	on offer
+		payment 250000
+		dialog `You recieve a message from Ivan after you land. "Thanks for all your help recently. We brought our findings to the Deep government and they gave us more funding! We've transfered 250,000 credits to your account and informed the Deep government of your recent help in this immense discovery. Please meet us on <destination> if you wish to assist us further. The help would be appreciated, b ...
```
\`You recieve a message from Ivan after you land. "Thanks for all your help recently. We brought our findings to the Deep government and they gave us more funding! We've transfered 250,000 credits to your account and made sure you were listed as a primary contributor on all the reports of these immense discoveries. Please meet us on \<destination\> if you wish to assist us further. Your help would be appreciated, but we will make progress even if you're a bit busy at the moment."\`

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+	on offer
+		conversation
+			`The scientists are able to quickly have the engines removed from your ship and ready in their research lab thanks to it being much closer to the spaceport. They pay you 417,000 credits for bringing the engines to them. After about three hours of analyzing the engines, Ivan walks up to you.`
+			`	"This is very interesting tech. They seem similar to human ion engines, but surprisingly better than even atomic engines, being both more powerful and more efficient at the same time. They're slightly more power-hungry, but it's still amazing that such an isolated group of humans could make such a technological leap in such a short amount of time."`
```
If the Remnant is friendly, they know the timeline. If hostile, they do not.

Amazinite replied:
Change `it's still amazing that such an isolated group of humans could make such a technological leap in such a short amount of time.` to `it's amazing that such an isolated group of humans could make technological leaps beyond what the Deep has done.`?

tehhowch replied:
I made some changes here on the PR i sent to you, see if you like what's there

tehhowch reviewed (commented):
Going to open a pull to you with a bunch of these changes done.

Amazinite commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1366 @@
+			names civilian
+			variant
+				"Freighter" 3
+				"Mule" 2
```
Then the fighters would also need to be saved.

tehhowch replied:
Not if they are added in a second NPC block :)

tehhowch replied:
Oops I forgot to do this in the last PR

Amazinite replied:
Got it. 👌

tehhowch committed:
Tehhowch Edits Round 2

* Text edits

Not sure about quoting some of the outfit names, but it seems appropriate in some cases.

* fixup! Text edits

* Removing comments and unwanted dialog

Amazinite committed:
Wording change

Amazinite committed:
Adding some notes

Amazinite committed:
Wording changes + fighter escorts

tehhowch commented on data/deep missions.txt:
```diff
@@ -81,6 +81,15 @@ mission "Deep: Syndicate Convoy"
+			names civilian
+			variant
+				"Dagger"
+			variant
```
They can both be in a single block.
If you want the chance for 0-2 of each:
```
npc
    personality timid
    government Merchant
    fleet
        fighters "civilian"
        variant
            Dagger 2
        variant
            Dagger
            Boxwing
        variant
            Boxwing 2
```

Amazinite replied:
Mules can only carry one fighter; there's only one Mule in this mission.

tehhowch replied:
Was thinking it was the one with 2 Mules

tehhowch commented on data/deep missions.txt:
```diff
@@ -343,6 +365,11 @@ mission "Deep: TMBR 0"
 				accept
+			
+			label decline
+			`	The group looks visibly dissapointed that you are unaware of the location of the band. A few minutes later, they inform you that they must depart for Midgard and say their farewells.`
```
disappointed

tehhowch commented on data/deep missions.txt:
```diff
@@ -394,7 +421,7 @@ mission "Deep: TMBR 2"
 			`	As Ulrich and the band take the stage, you spot the scientists in the front row, grinning and cheering wildly. The first song is one you heard them play in Hai space, and even though none in the audience know the words to sing along, they appear to love the message, and the cheers and screams only grow more frenetic.`
 			`	During an intermission, you invite the scientists onto your ship to personally meet the band. The thrill in the eyes of each of the scientists is astounding as they meet the members of "There Might Be Riots."`
-			`	Before the intermission ends, each of the scientists thank you personally for bringing the band to Midgard, and Isaac even hands you <payment>. "It's the least we could give you," he adds. As the scientists start to depart your ship to listen to the rest of the concert, Hannah kisses you on the cheek without a word before running off with the crowd.`
+			`	Before the intermission ends, each of the scientists thank you personally for bringing the band to Midgard, and Isaac even hands you <payment>. "It's the least we could give you," he adds. As the scientists start to depart your ship to listen to the rest of the concert, Hannah runs up and hugs you without a word before running off with the crowd.`
```
`Hannah runs back to hug you before rejoining her friends in the crowd`?

Amazinite committed:
The first mission of Mystery Cubes + the skeleton for the rest

Amazinite committed:
Wording fix

Amazinite commented on data/deep missions.txt:
```diff
@@ -273,29 +274,32 @@ mission "Deep: Kraz Convoy"
 		"deep convoy" ++
 		payment 1260000
-		dialog `When you return to <planet> with the freighters, you are greeted by a Deep Security officer who hands you <payment> and thanks you for accepting the job on such a short notice. "We'll make sure to contact you in the future should we need your help."
+		dialog `When you return to <planet> with the freighters, you are greeted by the same Deep Security officer that offered you the job. He hands you <payment>. "Thank you for accepting this job on such a short notice, Captain. I should introduce myself. I am Lieutenant Paris, and I will make sure to contact you in the future should the Deep require your help."`
```
I might turn this dialog into a conversation so that I can branch whether or not you've met Paris in Deep: Mystery Cubes 0, thus still allowing this mission to be done if you didn't do it before meeting Paris.

Amazinite committed:
Removing redundant condition

Part of starting the Deep: Remnant string involves having either 3
convoys done or finishing the Deep: TMBR string. Therefore, placing it
in an or structure with those exact two conditions is redudnant since
the precense of the has "Deep: Remnant Research: done" condition means
that at least of the other two conditions must already be true as well.

Amazinite committed:
Deep: Mystery Cubes 1 through 3

Writing late at night is fun.

Amazinite committed:
Typo fix

Amazinite committed:
Finishing Deep: Mystery Cubes

Amazinite commented:
Just a few more things to go.

In that most recent commit, I switched all the neutral pronouns for Beelzebub to masculine pronouns. Beelzebub will also be the captain of the Cosmic Devil and be behind the situation in the last missions. Truth is though, should Beelzebub make an appearance later on/in other missions, 1) Beelzebub is actually a woman (I have some interesting ideas on how that would be revealed) and 2) she won't actually be on the Cosmic Devil when it gets taken down.

Amazinite committed:
Typo fix

Amazinite committed:
Housekeeping + an uncomplete Deep: Singularity skeleton

I really have no idea what I'm going to do with the singularity missions
to make them different from the rest of the missions. None of the
missions so far involve transporting cargo so it might involve that, but
even then I don't know how to tie that in to the singularity.

Also, my deadline for finishing this up is this Wednesday, so I'm going
to try working on this all day tomorrow and hopefully get Deep:
Singularity completely done and start on rewriting Deep: Scientist
Rescue.

Amazinite committed:
Missed some lines in the last commit

tehhowch committed:
Tehhowch Edits Round 3

* Missing quotes and ill-formed condition fixes

* Removing two word label

It's not really something I've seen in the game files. And given that you can never know the label name without checking the game files, I don't really see the need to make the label more descriptive.

Amazinite commented:
The following is a draft for how I'm going to post the PR. Anything in brackets is a note.
<hr/>

This is a continuation of #2347 and addresses a number of my own critiques about my original attempt to license-lock the Bactrian as well as vastly expanding the amount of non-campaign story content in game (the amount of missions added by this PR almost rivals the number of missions currently in `transport missions.txt`).

The following is an overview of the various mission strings added by this PR which will lead up to the unlocking of the Bactrian:

* `Deep: Convoy`: Escort large convoys of merchants tasked with picking up materials from across the galaxy for the Deep.
   * Type: Escort
* `Deep: Mystery Cubes`: Help to hunt down the pirate warlord known only as Beelzebub, who is giving both Deep Security and the Republic Navy a headache.
   * Type: Combat
   * Notes: Tested in a single Corvette using only human technology [Not really yet]
* `Deep: TMBR`: Bring There Might Be Riots back from Hai space so they can play a concert in the Deep for a group of scientists that you meet.
   * Type: Transport
* `Deep: Remnant`: This mission string acts as a continuation of `Terminus exploration` as the scientist that you met in that mission continues to research the wormhole and what lies beyond it. Help the Deep learn about the existence of the Remnant, the human faction which technologically rivals even the Deep, and if you wish to, bring some Remnant technology to the Deep for them to analyze.
   * Type: Exploration
* `Deep: Singularity`: Be questioned by the Deep on your work for Albert Foster and learn more about the artificial singularity and the past of the Deep.
   * Type: [Anything but combat]
* `Deep: Scientist Rescue`: A rewritten version of the original mission string from #2347. The premise is the same, but it has been rewritten to include information and characters from the above mission strings that all lead up to this one.
   * Type: Combat
   * Note: Tested in a single Falcon using human and Hai technology [Not really yet]

All missions can be done without having to start the FW campaign, meaning that if someone really wanted to grind out spaceport missions and explore the galaxy early on then they could manage to unlock the Bactrian before starting the main plot, but otherwise I imagine that most people will unlock the Bactrian after the main plot has been completed.

Given the current setup, all of the following things would need to be done in order to unlock the Bactrian:
* 10 mystery cube jobs
* 3 transport scientist jobs
* 3 escort science vessel jobs
* 3 record academic conference jobs
* At least one of the three `Deep: Convoy` missions
* The `Deep: Mystery Cubes` string [5 missions]
* The `There Might Be Riots` string [4 missions] and `Deep: TMBR` strings [2 missions] 
  * (optional if the player instead does at least two of the three `Deep: Convoy` missions)
* The `Deep Archaeology` string [5 missions]
* The `Deep: Singularity` string
* The `Terminus exploration` mission
* The `Deep: Remnant` string [3 to 9 missions, depending on branch and if optional missions are completed]
* Discover the Hai and Remnant
* The `Deep: Scientist Rescue` string [3 missions]
<hr/>
Special thanks to @tehhowch who helped by rewriting some of my work.

Amazinite committed:
Draft of the first Deep: Singularity mission

Amazinite commented:
I have a few tests coming up, which means less homework. Going to finish at least Deep: Singularity over the weekend.

tehhowch commented:
Is there anything here that you think would benefit from having travel_to
available? And the boarding mission fixes as well.

On Sep 7, 2017 7:05 PM, "Amazinite" <notifications@github.com> wrote:

> I have a few tests coming up, which means less homework. Going to finish
> at least Deep: Singularity over the weekend.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tehhowch/endless-sky/pull/10#issuecomment-327962939>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AT54shbRpKWU6E31gjZEcL5PWFkBW7Knks5sgITcgaJpZM4OpEUc>
> .
>

Amazinite commented:
The end of Deep: Mystery Cubes could use travel_to, as currently the reinforcement fleet that spawns in Sol would follow the player if the player didn't go straight to Farpoint.

As for the boarding mission fixes, I might be able to tie those in to the Deep: Singularity missions.

tehhowch commented:
Of note is that no boarding mission will spawn from a mission NPC (which really complicates trying to make missions happen from a certain special derelict). I haven't tried it yet, but we may be able to create a custom fleet that is only a derelict ship, and use mission events to add it to system fleets. I'll report back on if this is possible.

(it is not possible to use the normal system fleet spawns to create derelict ships - Fleet class doesn't call ship->FinishLoading when doing the Fleet::Enter stuff, so even though they get the personality, they don't get disabled.)

Turtleroku commented:
I've been working on a different set of missions to unlock the Bactrian. It seems that acquiring such a powerful vessel so early without any cost is less rewarding than if there was some sort of cost. To me there is a small bit of lore that is lacking since colonization would probably require more than one ship design. To that end I wrote a three part story.

Turtleroku commented:
I have a three part story to unlock the bactrian. I have already written the dialog and now am working on formatting. Should I post my whole .txt file here or just a summary?

Where should I post questions to troubleshoot a problem?

tehhowch commented:
@Turtleroku the Discord's content-creating room is a great way to get (near) realtime feedback and help on mission content and formatting. A link to join the Discord community is: https://discord.gg/u6KepgG
To share your story with @Amazinite, myself, and others, you can either post it here, or (preferably) link us to particular branch from your repository fork (you have two separate repositories that seem to be the same). 
As for the current state of this branch, the opening chapters are not likely to change unless instructed by MZ when the branch is PR'd to the official repository. @Amazinite is still seeking ideas and input for the "Singularity" stage of the unlock story, which is currently a non-combat segment in which the player learns more about the early development of the Deep and the artificial singularity that was observed in the Deep: Archaeology mission strings, and other plot points that seem fitting - such as character development with Lt. Paris and others.

The existing mission of yours (https://github.com/Turtleroku/Bactrian-Mission/commit/bd7aa48ed3bf7ee9c97ac636ad0fa22e2209df45) is in theory similar to the final chapter in this branch - a search-and-rescue + fight-the-pirates mission string - but without seeing more of your story no further comparisons can be made.

Amazinite committed:
A rewrite of the Deep Scientists string

Amazinite committed:
Finished my rewrite of Scientist Rescue

Now I just need to figure out what the heck to write for Deep:
Singularity

Amazinite commented:
Done rewriting Scientist Rescue. Now it's just a matter of finding out what to write for Deep: Singularity, play testing, adding necessary logs, and PRing.

Amazinite committed:
Removing some needless branches

Amazinite committed:
Shortening up Mystery Cubes 2

Amazinite committed:
Finished the two preliminary Deep: Singularity missions for the string

Also wrote up a rough outline for the rest of the string; finally going
to get this done tomorrow, where I'll then do a play-test run and make
any final dialog changes before I PR.

Amazinite commented:
I wrote most of the last mission last night. I just have to finish up the last half of the on complete conversation as well as some on stopover conversations, then this will be essentially done. After that I'll play test it a few times (gave myself a lot to test given the branching nature of all these missions) and add any finishing touches to the dialog before I PR, which should end up being tonight or tomorrow afternoon depending on if I'm able to play test everything I want to hit on.

Going to PR without logs since that's just extra content, and the only missions with logs right now anyways are the main campaign missions.

tehhowch commented on data/deep jobs.txt:
```diff
@@ -80,12 +82,13 @@ mission "Deep Mystery Cube [2]"
 		system
 			distance 6 9
-		personality nemesis disables waiting
+		personality nemesis disables waiting plunders
```
Possibly also 'harvests'

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1914 @@
+	stopover Hephaestus
+	deadline
+	to offer
+		"combat rating" > 50
```
These are pretty low combat ratings IMO

Amazinite replied:
Probably. I'll tweak them after play testing.

Amazinite replied:
Payments may need tweaking too.

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1914 @@
+				"Lance" 4
+				"Frigate" 2
+	npc evade
+		personality staying heroic marked
```
Marked will mean that all the NPC allies won't help - the player alone has to defeat these ships.
Did you mean target?
My latest commit addresses this

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1914 @@
+			`You are approached by a man in a Deep Security uniform who asks you if you would be willing to escort a few freighters to <stopovers> and back by <date>. "We would normally have our own fleet do it," he says, "but we're short on available ships given certain circumstances."`
+			choice
+				`	"Sure, I can do that."`
+				`	"Sorry, but I will not be traveling in that direction any time soon."`
```
`I will not be` doesn't really mesh with the previous option's casual nature. Maybe
`I'm not`
or
`I won't be`?
My latest commit addresses this.

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,1914 @@
+	source
+		attributes deep
+	destination Memory
+	stopover "New Sahara"
```
Given the length of this mission, we could write a repeatable side mission that spawns some pirates randomly, while this mission (or perhaps some others in this string) are active, e.g.
```
mission "Pirate Patrol"
    invisible
    name "Defeat pirates"
    to offer
        has "Deep: Mystery Cubes 2: active"
        random < 10
    to fail
        not "Deep: Mystery Cubes 2: active"
    npc kill
        ...
```

Amazinite replied:
Makes sense. I'll look into doing that.

Amazinite committed:
Part of Deep: Singularity 1

Turtleroku commented:
I used ancient Egyptian gods for lots of the ships. 
I finished my short story. The last thing would be to blender model two ships but I don't think that's necessary right now. Here is the github link to the plugin's files.
https://github.com/Turtleroku/Bactrian-Mission

tehhowch committed:
Tehhowch Edits Round 4

* Style + text edits + more pirates + personality fix

 - Fixed indented conversation opener
 - Dropped closing " when the next paragraph was a continued quote from the same speaker
 - Swapped `marked` for `target`
 - Swapped npc definitions to be gov - personality - system - fleet
 - Tweaked some dialogues
 - Added filler pirates for if the player destroys the Left/Right Hand and lands on Haven instead of returning to Valhalla when finding the Star Queen.
 - Moves the stopover dialogs to separate missions, since the game doesn't support individual stopover dialogs, just a final "all stopovers are now done" trigger

* Changed random offer for Singularity 0B to timed trigger

Amazinite committed:
Merge branch 'master' into deep-missions

Amazinite committed:
Putting plunders back on the bounty hunters

tehhowch commented:
@Turtleroku your plugin's dialogue need a fair bit of touch-up - for example, many of your questions do not end in "?", but rather ".", which is quite incorrect. 
Various offer statements and conditions don't seem to be well-formatted - have you playtested every choice to ensure it works correctly? You also have unsupported event actions - you cannot `remove object`. I suspect if you review your "errors.txt" file you will find quite a few additional things that you need to address.

Turtleroku commented:
Yes I have playtested every choice and it works, (the only thing that might not work in the payments in dialog). Where is the error.txt? 
I believe that remove object does work, if I take that out of the code the system remains colored (habitable) and the planet can still be viewed from the map.
edit: I looked through it, and fixed all the incorrect "." vs "?" 
edit: found the errors.txt (it was staring me in the face), fixed the errors, thanks for the suggestion tehhowch)

Turtleroku commented:
I don't know if my short story could be incorporated into what has already been written either before, after or middle. I view the Bactrian as an awesome ship but not an achievement medal, so I didn't want to make the process too long.
So you don't have to play it to understand the plot of my story I attached a synopsis. Spoilers
[story outline(spoilers).txt](https://github.com/tehhowch/endless-sky/files/1368600/story.outline.spoilers.txt)

Amazinite committed:
Finishing the last of the Deep missions

Amazinite commented:
@Turtleroku Given that the work on this PR has been storyboarded for 2.5 months now and was just finished, there's no way I'll be able to fit your missions in. And as @tehhowch stated previously, the general concept of looking for a ship is already covered in the final missions of this PR, meaning that to add your missions would be somewhat redundant.

Edit: If you want to pursue getting your missions into the game though, you could go ahead and independently PR your missions to the game and get feedback from a wider audience.

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,2219 @@
+				`	"You can do it!"`
+				
+			`	Hannah slams the virtual tennis ball with her paddle, causing it to bounce off the table and fly right past the hologram opponent before disinegrating into thin air.`
+			`	"I did pretty good, di
```
Should these scanned fleets receive `target`? If so, we can drop the clear/reset events entirely, as it will be apparent which ships should get scanned.

Amazinite replied:
I'm guessing you meant to comment this under the Remnant missions; sure, that makes sense.

tehhowch replied:
yeah.. thought it was there... at least it was on my screen when i made it... lol

tehhowch commented on data/deep missions.txt:
```diff
@@ -0,0 +1,2219 @@
+					defer
+					
+			`	You approach the Bounder and count five people dancing in total: three women and two men. One of the women dancing notices you walking over and exclaims, "Looks like someone else is about to join the party!" Her shout is met by a raucous cheer from the rest of the dancers, and they pull you into the fray.`
+			`	After a few minutes of dancing, the music begins to subside. One of the dancers introduces himself to you as Garrison, and lists off the names of his fellow dancers; Isaac, Maria, Hannah, and Laura. As you make your introductions, you learn that everyone here is part of the same research team, working for one of the Deep's many science bureaus, and they are currently wrapping up a short vaca ...
```
We have both `Ivan` and `Isaac` as separate scientist characters - someone might need a name change.
Top picks:
Pierre
Niels (or Niles)
or some other non- `I` name would work.

Amazinite replied:
If necessary, Isaac -> Pierre is fine by me,

tehhowch committed:
Tehhowch Edits 5

* Updates

 - Swapped some NPC orders to make the flagship the cargo ship
 - Convo edits
 - Quoted government names
 - Isaac -> Pierre, to reduce Ivan/Isaac confusion potential
 - Updated the `near` restrictions
 - Added some attributes to ensure missions which reference spaceports but are `landing` don't offer on weird places.
 - D.S.S. -> D.P.S. , as this matches the signage on the ships in the Deep Archaeology string.

* Add feedback

Amazinite committed:
Fixes, rebalancing, and wording changes

tehhowch committed:
Use the new `not` filter

- Better prevention against missions starting on stations if they have the player interacting in manners not expected for space stations.

tehhowch commented on data/deep missions.txt:
```diff
@@ -409,8 +410,11 @@ mission "Deep: Mystery Cubes 2"
 			`	In the end, only one of the merchant captains declines. She is escorted out of the room, and the admiral explains each merchant captain's new role.`
 			`	"We assume that Beelzebub knows where we will be, but we know where he is as well - the stolen Cruisers and the pirate fleet that attacked our base moved toward the Core. Given the severity of the crime that Beelzebub has just committed, he will undoubtedly attempt to get as far away from the law as possible. That leaves us only two possible destinations: Buccaneer Bay, or the anarchist colo ...
-			`	"We'll be setting up multiple blockades around the galaxy in hopes of boxing Beelzebub in as we sweep the galaxy. Captain <last>, you will be stationed on <destination> with Lieutenant Paris and a small Navy fleet. Please gather your things and travel there immediately."`
+			`	"We'll be setting up multiple blockades in hopes of boxing Beelzebub in as a number of fleets sweep the galaxy. Captain <last>, you will be stationed on <destination> with Lieutenant Paris and a small Navy fleet. Please gather your things and travel there immediately."`
```
For something someone is saying, especially a high-ranked commanding officer, i think "we" is better.

tehhowch commented on data/deep missions.txt:
```diff
@@ -491,9 +495,9 @@ mission "Deep: Mystery Cubes 3"
+			`	A rather young recruit asks, "Should we go help them?" to which a senior officer responds, "We were given orders and we're going to follow them unless told otherwise." The comms fall silent, and the waiting resumes."`
 			``
-			`	A few anxious hours pass, with no new information. Without warning though, one of the Ravens takes off and your communications device begins blaring. "A pirate fleet just jumped into the system, alongside two Navy Cruisers!" All the captains and their crew scatter for their ships, ready to fight.`
+			`	A few anxious hours pass, with no new information. Without warning though, one of the Ravens takes off and your communications device begins blaring. "A pirate fleet just jumped into the system, alongside two Navy Cruisers painted in pirate colors!" All the captains and their crew scatter for their ships, ready to fight.`
```
`repainted Navy Cruisers` is maybe more conversational, and less duplicative, as they lead with "a pirate fleet" so it's expected the colors are pirate colors

Amazinite committed:
Payment and pirate tweaks

tehhowch committed:
Personalities

Amazinite committed:
Scientist Rescue combat balance

Amazinite committed:
Arneb fleets reset to normal if player fails missions

tehhowch commented:
closing since endless-sky#3120 is now a thing :+1:
